CI is failing

Just to keep a trace of that failure.

##  System Information

### Linux distribution

``` Ubuntu Xenial ```

### Terraform version

```sh
terraform_0.12.0_linux_amd64
```
## Description of Issue/Question

### Steps to Reproduce Issue

Run a Travis build: 

```
go: git.apache.org/thrift.git@v0.0.0-20180902110319-2566ecd5d999: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/travis/gopath/pkg/mod/cache/vcs/83dba939f95a790e497d565fc4418400145a1a514f955fa052f662d56e920c3e: exit status 128:
	fatal: unable to access 'https://git.apache.org/thrift.git/': Failed to connect to git.apache.org port 443: Connection timed out
go: error loading module requirements
Makefile:45: recipe for target 'fmt-check' failed
make: *** [fmt-check] Error 1
The command "make fmt-check" exited with 2.
```
___
## Additional information:

Apache seems to mirror its git.apache.org with github.com. We could replace `git.apache.org/thrift.git` to `github.com/apache/thrift` in `go.mod` or we could wait for git.apache.org to be up & running again. (https://status.apache.org/incidents/63030p4241xj?u=06cfnjqp03yp)

--- a/go.mod
+++ b/go.mod
@@ -25,3 +25,5 @@ require (
 	golang.org/x/lint v0.0.0-20181217174547-8f45f776aaf1
 	golang.org/x/tools v0.0.0-20181219222714-6e267b5cc78e // indirect
 )
+
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999

